### DISCUSSION TITLE
Make it piossible to import multiple files at once

### DISCUSSION THREAD

**Discussion by LuposX** [ppy/osu#23044] [Ideas]
At the moment if you want to import multiple files you need to import them all sperately, it would be nice to lcik a folder and all files in the folder will be imported.

**Comment by Theighlin** [comment]
Clicking a file is different than clicking a folder, unless you mean it is zipped and contains only osu-readable files.
Anyway, you can import multiple files already by selecting them all at once, and dragging them into the game. (Contrary to popular belief, you can drag and drop while in fullscreen via alt-tab, at least on windows)

**Comment by peppy** [comment]
Where are you hoping to "click" to make the import start?
